scr/main/java/bittercode/model/User.java

```diff
@@ -0,0 +1,88 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class User implements Serializable {
+
+    private String emailId;
+    private String password;
+    private String firstName;
+    private String lastName;
+    private Long phone;
+    private Char address;
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Possible incorrect data type for address</b></div>
<div id="fix">

The `address` field is declared as `Char`, which might not be suitable for storing an address. Consider using `String` instead for more flexibility in storing address information. A similar issue was also found in scr/main/java/bittercode/model/User.java (line 56-58).
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -13,1 +13,1 @@
 -    private Char address;
 +    private String address;
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/3#issuecomment-2434572244>#7fa0a4</a></i></small>
</div>


---
🚩 Problem result:
- [ ] This issue was flagged incorrectly